tests/common/matchers/Matchers.h~

```diff
@@ -21,9 +21,21 @@
 #define MATCHERS_H_
```

Comment:
What is that file? It has ~ at the end of its name. Temporary vim file (?)

Reply:
No, it's an emacs (my editor) backup file. Well spotted.

I'll add something to git ignore instructions to prevent this in the future